lib/langchain/assistants/llm/adapters/aws_bedrock.rb

```diff
@@ -0,0 +1,10 @@
+module Langchain
+  class Assistant
+    module LLM
+      module Adapters
+        class AwsBedrock < Anthropic
```

Comment:
Is this adapter only going to work with the Anthropic models hosted on AWS Bedrock? Aren't there other chat models that are accessible on AWS Bedrock?

Reply:
Yeah, there are numerous chat models available on AWS Bedrock. However, the `LLM::AwsBedrock#chat` currently only supports the Anthropic models. I attempted to use the `switch...case` approach to support models other than the Anthropic ones, but it isn't feasible ATM.

In the future, we may consider implementing the `LLM::AwsBedrock#converse` to accommodate all chat models for the assistant instead of the `LLM::AwsBedrock#chat` (invoke_model).